projects/core/src/global-message/config/default-global-message-config.ts

```diff
@@ -0,0 +1,15 @@
+import { GlobalMessageConfig } from './global-message-config';
+import { GlobalMessageType } from '../models/global-message.model';
+
+export function defaultGlobalMessageConfigFactory(): GlobalMessageConfig {
+  return {
+    globalMessages: {
+      [GlobalMessageType.MSG_TYPE_CONFIRMATION]: {
+        timeout: 3000,
```

Comment:
Just curious, any specific reasons behind those values? :)

Reply:
no reason, it's just sample data